SFR/Assembly/inner_assembly/make_SFR_assembly_model.py

```diff
@@ -0,0 +1,209 @@
+import openmc 
+import numpy as np
+from argparse import ArgumentParser
+import openmc.material 
+import warnings
+warnings.filterwarnings('ignore')
+
+
+# Create the argument parser
+ap = ArgumentParser(description="SFR Pincell Model Generator")
+ap.add_argument('-n', dest='n_axial', type=int, default=100, help='Number of cells in the Z direction')
+ap.add_argument('-e', '--shannon_entropy', action='store_true', help='Add Shannon entropy mesh')
+ap.add_argument('--height', dest='height_of_the_core', type=float, default=30.0, help='Height of the reactor core')
+
+# Parse the arguments
+arguments = ap.parse_args()
+
+arguments=ap.parse_args()
+
+N=arguments.n_axial
+height=arguments.height_of_the_core
+shannon_entropy=arguments.shannon_entropy
+
+
+###################################################################
+##########################   Materials     ########################
+###################################################################
```

Comment:
Let's consider making a function that returns a dictionary of materials `create_SFR_materials()` and converting use of the material objects to material_dict["mat_name"]